web/modules/weather_i18n/translations/en.po

```diff
@@ -0,0 +1,366 @@
+msgid "Intensity"
+msgstr ""
+
+#: wind.labels.speed-from-direction
+msgid "@speed mph from the @direction"
+msgstr ""
+
+#: loader.loading-text
+msgid "Loading the forecast"
+msgstr ""
+
+#: wfo.labels.facebook
+msgid "Facebook"
+msgstr ""
+
+#: wfo.labels.phone
+msgid "Phone"
+msgstr ""
+
+#: wfo.labels.twitter
+msgid "X"
+msgstr ""
+
+#: wfo.labels.youtube
+msgid "YouTube"
+msgstr ""
+
+#:footer.aria.noaa-logo
+msgid "National Oceanic and Atmospheric Administration logo"
+msgstr ""
+
+#: footer.noaa-motto
+msgid "Science. Service. Stewardship"
+msgstr ""
+
+#: uswds-banner.skip-to-content
+msgid "Skip to main content"
+msgstr ""
+
+#: forecast.labels.weather-info
+msgid "Weather information sections"
+msgstr ""
+
+#: forecast.labels.alerts
+msgid "Alerts"
+msgstr ""
+
+#: forecast.labels.current
+msgid "Current"
+msgstr ""
+
+#: forecast.labels.seven-days
+msgid "7 Days"
+msgstr ""
+
+#: forecast.errors.current-conditions
+msgid "There was an error loading the current conditions."
+msgstr ""
+
+#: forecast.current.weather-as-of
+msgid "Weather as of"
+msgstr ""
+
+#: forecast.current.aria.feels-like
+msgid "It feels like @feelsLike 	."
+msgstr ""
+
+#: forecast.current.feels-like
+msgid "Feels like"
+msgstr ""
+
+#: forecast.current.obs-station
+msgid "Observation station"
+msgstr ""
+
+#: forecast.current.last-updated
+msgid "Last updated"
+msgstr ""
+
+#: forecast.errors.daily
+msgid "There was an error loading the daily forecast."
+msgstr ""
+
+#: daily-forecast.labels.today
+msgid "Today"
+msgstr ""
+
+#: safety.labels.tips
+msgid "Tips to stay safe"
+msgstr ""
+
+#: hourly-forecast.labels.forecast
+msgid "Forecast"
+msgstr ""
+
+#: forecast.errors.hourly
+msgid "There was an error loading the hourly forecast."
+msgstr ""
+
+#: precip-table.labels.amounts
+msgid "Precipitation amounts"
+msgstr ""
+
+#: precip-table.aria.amounts
+msgid "precipitation amounts for the coming hours"
+msgstr ""
+
+#: precip-table.aria.period
+msgid "time period"
+msgstr ""
+
+#: precip-table.aria.precipitation
+msgid "forecast precipitation"
+msgstr ""
+
+#: units.inches
+msgid "inches"
+msgstr ""
+
+#: alerts.text.in-effect-from
+msgid "In effect from"
+msgstr ""
+
+#: alerts.text.issued-by
+msgid "Issued by @sender"
+msgstr ""
+
+#: alerts.labels.what-to-do
+msgid "What to do"
+msgstr ""
+
+#: alerts.labels.areas-impacted
+msgid "Areas impacted"
+msgstr ""
+
+#: alerts.text.counties-in
+msgid "Counties in"
+msgstr ""
+
+#: alerts.text.including-cities
+msgid "Including these cities"
+msgstr ""
+
+#: location-search.labels.enter-city-zip
+msgid "Enter city or zip code"
+msgstr ""
+
+#: location-search.labels.use-location
+msgid "Use my location"
+msgstr ""
+
+#: forecast.labels.weather-story
+msgid "Forecaster's report"
+msgstr ""
+
+#: wfo.labels.local-office
+msgid "Your Local Forecast Office"
+msgstr ""
+
+#: wfo.labels.find-us
+msgid "Find us on"
+msgstr ""
+
+#: theme.text.submitted-by
+msgid "Submitted by @author_name on @date"
+msgstr ""
```

Comment:
```suggestion
msgstr ""

#: backend.login.sso-login-path
msgid "SSO Login Path"
msgstr ""

#: backend.login.login-text
msgid "Login Text"
msgstr ""

#: backend.login.sso-cancel-path
msgid "SSO Cancel Path"
msgstr ""

#: backend.login.cancel-text
msgid "Cancel Text
msgstr ""
```

These are using the `t()` function on the backend, for building up the SSO admin form. It's less critical that they be translated since it's in an admin form, but it'd still be good to do I guess?

Reply:
How did you find these in particular? When I used the `potx` tool, I found thousands of instances of translated strings on the backend/cms, and specifically excluded them from this file because it's just too much, and we don't necessarily care about translating the CMS interface at this point.
  
That said, I'm not especially against including these few in the file for now

Reply:
There is also this method ``{{ 'Some string' | t }}`` and in the core CMS templates (which we dont directly see unless we go looking) the `trans` filter is also used prodigiously. All of this makes it difficult to actually find what is being translated using normal string parsing -- I think using Twig modules themselves would make it easier. But the `potx` module _does_ use basic string parsing to get these all out